pkg/apis/batch/v2alpha1/types.go

```diff
@@ -116,6 +116,11 @@ type CronJobSpec struct {
 	// This is a pointer to distinguish between explicit zero and not specified.
 	// +optional
 	FailedJobsHistoryLimit *int32 `json:"failedJobsHistoryLimit,omitempty" protobuf:"varint,7,opt,name=failedJobsHistoryLimit"`
+
+	// The timezone to be used when scheduling the job
+	// Valid options include anything in the IANA Time Zone database
+	// +optional
+	Timezone *string `json:"timezone" protobuf:"bytes,8,opt,name=timezone"`
```

Comment:
The protobuf generator should handle it accordingly. The number (here 8) should stay, otherwise this will break backwards compatibility.